src/cloud-cli/meltano/cloud/api/client.py

```diff
@@ -244,18 +244,30 @@ async def run_project(
         Args:
             deployment: The name of the Meltano Cloud deployment in which to run.
             job_or_schedule: The name of the job or schedule to run.
+
+        Raises:
+            MeltanoCloudError: The Meltano Cloud API responded with an error.
         """
         async with self.authenticated():
-            url = (
-                "/run/v1/external/"
-                f"{self.config.tenant_resource_key}/"
-                f"{self.config.internal_project_id}/"
-                f"{deployment}/{job_or_schedule}"
-            )
-            await self._json_request(
-                "POST",
-                url,
-            )
+            try:
+                url = (
+                    "/run/v1/external/"
+                    f"{self.config.tenant_resource_key}/"
+                    f"{self.config.internal_project_id}/"
+                    f"{deployment}/{job_or_schedule}"
+                )
+                await self._json_request(
+                    "POST",
+                    url,
+                )
+            except MeltanoCloudError as ex:
+                if ex.response.status == 400:  # noqa: WPS432
```

Comment:
The response we're looking for from the runner would be a status code of `400`. Flakeheaven didn't like this though. Any issues with this @edgarrmondragon @WillDaSilva?

Reply:
Ah got it. Thanks!